src/agor/tools/subprocess_manager.py

```
@@ -0,0 +1,399 @@
+            })
+            
+            # Raise exception if check=True and command failed
+            if check and not success:
+                raise SubprocessError(
+                    f"Command failed with exit code {returncode}",
+                    cmd_list, returncode, stdout, stderr
+                )
+            
+            return success, stdout, stderr, returncode
+            
+        except subprocess.TimeoutExpired as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)